recipes/ophyd/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ophyd" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "74ebb49196067f86450142da1dc3732267d31e8e9b4d32125b673d8464ce38fb" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: ophyd-v{{ version }}.tar.gz
+  url: https://github.com/NSLS-II/ophyd/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
```

Comment:
I don't think you can use `noarch: python` if this is a py3 only package. It seems that you should be using the `skip: True  # [py2k]` selector.